js/ui/search.js

```diff
@@ -367,14 +398,58 @@ const GridSearchResults = new Lang.Class({
 });
 Signals.addSignalMethods(GridSearchResults.prototype);
 
+const SearchResultsBin = new Lang.Class({
+    Name: 'SearchResultsBin',
+    Extends: St.BoxLayout,
+
+    vfunc_get_preferred_height: function(forHeight) {
+        let themeNode = this.get_theme_node();
+        let marginBottom = themeNode.get_length('-margin-bottom');
+
+        let [min, nat] = this.parent(forHeight);
+        return [min + marginBottom, nat + marginBottom];
+    },
+
+    vfunc_get_preferred_width: function(forHeight) {
+        let themeNode = this.get_theme_node();
+        let maxWidth = themeNode.get_max_width();
+        let marginHorizontal = themeNode.get_length('-margin-horizontal');
+
+        let [min, nat] = this.parent(forHeight);
+        return [Math.min(maxWidth, min) + marginHorizontal,
```

Comment:
Again, not my code. But I agree with you it's probably better to remove this if possible